main/main.cpp

```diff
@@ -1649,6 +1651,8 @@ void Main::cleanup() {
 		memdelete(script_debugger);
 	}
 
+	Color::_clear_named();
```

Comment:
Maybe this could go in `unregister_core_types()` instead? It's already where `CoreStringNames` and `StringName` gets cleaned up.

Reply:
Eh, didn't notice we already have a place for those, going to move it there :)

Reply:
Ping me when it's updated, I'm waiting for this last fix to build 2.1.5 RC2.